windows/windowsservercore-ltsc2019/Dockerfile

```
@@ -23,7 +23,13 @@
 #  THE SOFTWARE.
 
 ARG JAVA_VERSION=17.0.7_7
-FROM eclipse-temurin:"${JAVA_VERSION}"-jdk-windowsservercore-1809
+FROM eclipse-temurin:"${JAVA_VERSION}"-jdk-windowsservercore-1809 AS jdk-core
+
+FROM mcr.microsoft.com/powershell:windowsservercore-1809
+
+ARG JAVA_HOME="C:\openjdk-17"
```

```suggestion
ARG JAVA_HOME="C:\openjdk-17"
ENV JAVA_HOME=${JAVA_HOME}
```

The `eclipse-temurin` provides the `%JAVA_HOME%` environment variable to consumers: let's keep it :)